File: src/gerente_relaciones.py
"""Módulo que define la clase GerenteRelaciones."""

from src.usuario import Usuario


class GerenteRelaciones(Usuario):
    """Representa al gerente de relaciones de la empresa."""

    def __init__(self, id_usuario: int, nombre: str,
                 email: str, contrasena: str):
        super().__init__(id_usuario, nombre, email, contrasena)

    def login(self) -> bool:
        print(f"{self.get_nombre()} ha iniciado sesión.")
        return True

    def logout(self) -> None:
        print(f"{self.get_nombre()} ha cerrado sesion.")

    def recibir_queja(self) -> None:
        print(f"{self.get_nombre()} ha recibido una queja.")

File: Televentas Java/Clases/GerenteRelaciones.java
package Clases;

public class GerenteRelaciones extends Usuario {

    public GerenteRelaciones(int id, String nombre, String email, String contrasena) {
        super(id, nombre, email, contrasena);
    }

    @Override
    public boolean login() {
        System.out.println(getNombre() + " ha iniciado sesión.");
        return true;
    }

    @Override
    public void logout() {
        System.out.println(getNombre() + " ha cerrado sesion.");
    }

    public void recibirQueja() {
        System.out.println(getNombre() + " ha recibido una queja.");
    }
}
